flashcards/backend/decks/tests/test_pagination.py

```diff
@@ -0,0 +1,15 @@
+from decks import factories
+from decks.pagination import DecksPagination
+from decks.viewsets import DeckViewSet
+
+
+def test_deck_list_pagination(api_rf, db):
```

Comment:
correct me if I'm wrong but you use api_rf fixture which comes from quiz app, can we use fixtures from another apps?

Reply:
Nope

Reply:
I have changed it to correct fixture now. Thanks